coding-concepts.js

```diff
@@ -9,51 +9,50 @@
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+
+// a) Your answer: the output that we would expect is that the string "indigo" would be added at the end of the array.
+// b) Verify and explain: Because I did not call on the variable "color" I did not get the expected outcome. All it did was push out the number of 5 which correlates to the number of index.
```

Comment:
Push definitely returns in a way that is often contrary to what we would assume.  Push returns the length of the new array.